README.md

```diff
@@ -1,145 +1,105 @@
-# Planner
-
-A tool to help manage [codebar.io](http://codebar.io) members and events.
+# codebar website & workshop planner
 
 [![Build Status](https://travis-ci.org/codebar/planner.png?branch=master)](https://travis-ci.org/codebar/planner)
 [![Coverage Status](https://coveralls.io/repos/codebar/planner/badge.png)](https://coveralls.io/r/codebar/planner)
 [![Code Climate](https://codeclimate.com/github/codebar/planner.png)](https://codeclimate.com/github/codebar/planner)
 
-If you are considering making a PR, please take a look at the [GitHub issues](https://github.com/codebar/planner/issues) to see if there are any new feature requirements or bugs that you maybe able to help resolve.
+A tool to help manage [codebar.io](http://codebar.io) members and events.
```

Comment:
That change is not part of this PR, I just changed the order of the lines, I didn't add the link. I can add that change later but I don't have time right now, so if we want this shipped sooner than later then it might be better as a separate issue.

Reply:
@KimberleyCook Update: Done (via the web interface). I also fixed an invalid Slack URL.